<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>1280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>256</y>
     <width>720</width>
     <height>10</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>512</y>
     <width>720</width>
     <height>10</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1024</y>
     <width>720</width>
     <height>10</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>768</y>
     <width>720</width>
     <height>10</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_acc">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>241</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
	 <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Accel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>0</y>
     <width>10</width>
     <height>1280</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>85</y>
     <width>485</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>485</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>485</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>485</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>595</y>
     <width>485</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>680</y>
     <width>485</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_gyo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>241</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
	 <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Gyro</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_lig">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>860</y>
     <width>241</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
	 <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Light</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_pro">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1120</y>
     <width>241</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
	 <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Prox</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_acc_x">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>471</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
	 <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_acc_y">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>90</y>
     <width>481</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
	 <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_acc_z">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>180</y>
     <width>481</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
	 <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mag">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>610</y>
     <width>241</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
	 <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Mag</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_gyo_z">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>440</y>
     <width>481</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
	 <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gyo_y">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>350</y>
     <width>481</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
	 <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gyo_x">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>260</y>
     <width>481</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
	 <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mag_z">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>690</y>
     <width>481</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
	 <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mag_y">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>600</y>
     <width>481</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
	 <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mag_x">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>520</y>
     <width>481</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
	 <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_lig_x">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>860</y>
     <width>481</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
	 <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pro_x">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>1130</y>
     <width>481</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
	 <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Near</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Exit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>1190</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
